privaterelay/locales/config/pontoon.toml

```diff
@@ -0,0 +1,15 @@
+# All files translated by the community by at least some locales should
+# be covered by the [[paths]] in this file.
+
+basepath = ".."
+
+locales = [
+    "de",
+    "en-CA",
+    "en-GB",
+    "fr",
+]
```

Comment:
No – This is where I think we set which languages we want to support. We need @peiying2 to take a look to see how we need to actually configure this before merging. 

@peiying2 Could you take a quick look to make sure we've set this up correctly? Additionally, if this needs to stay OUT of Pontoon for now, please advise if we need to remove anything.

Reply:
If this project is not ready for Pontoon, then can we exclude the file? Or since the project is not set up in Pontoon,  so nothing will be picked up there, and you can keep your work here.

Reply:
A side question, is this considered marketing  content, thus Anika and the like would be working on this once enabled in Pontoon. Or is it for the communities?

Reply:
> A side question, is this considered marketing content, thus Anika and the like would be working on this once enabled in Pontoon. Or is it for the communities?

I would assume it would be treated the same as [Monitor](https://pontoon.mozilla.org/projects/firefox-monitor-website/) or [Facebook Container](https://pontoon.mozilla.org/projects/facebook-container/)

Reply:
@maxxcrawford - I would say that we should pull this list from an env var like we do for Monitor, but that just shifts the responsibility from an app code change to an ops code change, so I guess 🤷‍♂️ . Maybe best to keep it in our own code.

Reply:
@peiying2 Excellent. Thanks for the insight on this.  

@groovecoder I recommend we remove the file all together and let it supported locales be set in Pontoon.

Reply:
This file has been removed.